src/searchengine/nova3/novaprinter.py

```diff
@@ -26,12 +26,16 @@
 
 
 def prettyPrinter(dictionary):
-    dictionary['size'] = anySizeToBytes(dictionary['size'])
-    outtext = "|".join((dictionary["link"], dictionary["name"].replace("|", " "),
-                        str(dictionary["size"]), str(dictionary["seeds"]),
-                        str(dictionary["leech"]), dictionary["engine_url"]))
-    if 'desc_link' in dictionary:
-        outtext = "|".join((outtext, dictionary["desc_link"]))
+    outtext = "|".join((
```

Comment:
The `VERSION:` on top of this file should be incremented.

Reply:
You're right! I'm guessing `1.48` is fine? Or should it be 2.0 since it adds fields to the output?

Reply:
According to the history it's always been bumped by 0.1 even when format changes slightly, so I'll do the same.